projects/module-1/hello-worlds.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Hello Worlds - Pair
+---
+
+## Learning Goals
+
+* Write clean, DRY JavaScript
+  * Build out functionality using functions that show trends toward SRP
+  * Manipulate the page after it has loaded adding, removing, and updating elements on the DOM
+* Begin to understand the connection between HTML, CSS and JavaScript
+  * Practice reading, understanding, and using provided code
+* Build an understanding of writing code collaboratively
+  * Document changes with atomic commits & thorough code reviews
+  * Communicate, troubleshoot, and plan effectively as a team  
+  * Ensure all team members are able to be heard and contribute throughout the project
+
+## Overview
+
+"Hello World" is a classic programming phrase with a [long history](https://www.hackerrank.com/blog/the-history-of-hello-world/). It's become an affectionate trope that, when learning a new programming language, one of the first things you learn to do is print "Hello World" - to a console, to a terminal, to a browser page. 
+
+We're turning this into a game about language! Our users will guess what language the phrase "Hello world!" appears in, and we'll keep track of their guesses.
+
+## Resources 
+
+<section class="answer">
+### Helpful Hints
+
+<section class="answer">
+### What's the deal with deploying?
```

Comment:
This section is at the beginning of the project spec, which could make a student think that it is critical for their project form the start, but I see deployment as a nice to have at the end. Is this something that would make send to be moved toward the end of the spec?

Reply:
Great idea! I will shift this down below the extensions section.